data_model.md

```diff
@@ -250,13 +243,13 @@ of forecast parameters):
 ### Aggregates
 {: .anchor}
 
-Each Aggregate must be associated with one or more Observations. Solar
-Forecast Arbiter will calculate aggregated observation data generated
-from these Observations. Each aggregate will also provide a list of
+Each Aggregate must be associated with one or more Observations. The Solar
+Forecast Arbiter calculates aggregated observation data
+from these Observations. Each aggregate also provides a list of
 Sites associated with the Observations. These Sites will provide
 forecast providers with the modeling parameters for each system in the
-Aggregate. Aggregate forecasts will be associated directly with an
-Aggregate. Forecast providers will be required to submit a forecast of
+Aggregate. Aggregate forecasts are directly associated with an
+Aggregate. Forecast providers are required to submit a forecast of
 the aggregated quantity.
 
 Aggregates have the following required parameters:
```

Comment:
@alorenzo175 should I update these parameters to match https://api.solarforecastarbiter.org/#tag/Aggregates/paths/~1aggregates~1/post ?

Reply:
yes